lib/fluent/agent.rb

```
@@ -160,9 +160,11 @@ def add_filter(type, pattern, conf)
 
     # For handling invalid record
     def emit_error_event(tag, time, record, error)
+      
     end
 
     def handle_emits_error(tag, es, error)
+     
     end
```

RuboCop (1.75.5)

[convention] Put empty method definitions on a single line.

(Style/EmptyMethod)